test/services/user/services/userService.test.js

```diff
@@ -83,6 +83,14 @@ describe('user service', () => {
 			});
 	});
 
+	it('importHash will set set, when creating a new user', async () => {
+		const student = await app.service('users').create({
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['student' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5424589)